workflow-templates/release-on-milestone-closed.workflow.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "Release on milestone closed",
```

Comment:
What's this file for? It's not in referenced example and I don't see it in GH docs either